share/functions/help.fish

```diff
@@ -86,6 +86,14 @@ function help --description 'Show help for the fish shell'
                 set fish_browser $cmd[1]
```

Comment:
^^^ The line above this should simply include `powershell.exe`.

Reply:
(github doesn't allow me to put a comment there because it's too far from a changed part of the diff)

Reply:
Ahh yea apologies for the misunderstanding here. 

This makes the diff much smaller and keeps everything where it should be - glossed over that block